be/Cf.WebApi/Endpoints/UserReviewEndpoints.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Cf.Application.Services.Interfaces;
+using Cf.Domain.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Cf.WebApi.Routing;
+using Cf.Contracts.Responses;
+
+namespace Cf.WebApi.Endpoints;
+
+public static class UserReviewEndpoints
+{
+    private const string Tag = "UserReviews";
+    private const string GroupName = "userReviews";
+
+    public static void MapUserReviewRoutes(this IEndpointRouteBuilder builder)
+    {
+        var group = builder
+        .MapGroup(GroupName)
+        .WithTags(Tag)
+        .HasApiVersion(1);
+
+        group.MapPost(CreateAsync);
+        group.MapGet(GetByServiceId);
+    }
+
+    [Authorize(Roles = "User")]
+    private static async Task CreateAsync([FromServices] IUserReviewService service, Guid jobId, ReviewModel model, IHttpContextAccessor httpContextAccessor) =>
```

Comment:
Correct end-point, jobId will come from `[FromRoute]`.

Reply:
It looks to me like it understands that by default here. But okay I can add this to be more specified

Reply:
![image](https://github.com/MatasAdmoneo/CarFans/assets/120173596/3dba71b1-5e2b-43ae-bc51-8adc53c4331e)

Reply:
Maybe you forgot to map the naming, example below:
![image](https://github.com/MatasAdmoneo/CarFans/assets/80633152/9bfa83da-522e-43c6-a76d-40d7b18a831b)

Reply:
Hmm that maybe be